src/CreeDictionary/API/search/test_eip_crk.py

```diff
@@ -1,24 +1,72 @@
+import pytest
+
 from CreeDictionary.API.models import Wordform
 from CreeDictionary.API.search.core import SearchRun
 from CreeDictionary.API.search.eip import EipSearch
 from CreeDictionary.API.search.types import Result
 
 
-def test_eip_search(db):
-    search_run = SearchRun("they crawled")
+@pytest.mark.parametrize(
+    ("search", "params"),
+    [
+        [
+            "they crawled",
+            {
+                "expected_query_terms": ["crawls"],
+                "expected_new_tags": ["+V", "+AI", "PV/ki+", "+Ind", "+3Pl"],
+                "lemma": "pimitâcimow",
+                "expected_inflection": "kî-pimitâcimowak",
+            },
+        ],
+        [
+            "they saw me",
+            {
+                "expected_query_terms": ["see"],
+                "expected_new_tags": ["+V", "+TA", "PV/ki+", "+Ind", "+3Pl", "+1SgO"],
+                "lemma": "wâpamêw",
+                "expected_inflection": "nikî-wâpamikwak",
+            },
+        ],
+        [
+            "my little bears",
+            {
+                "expected_query_terms": ["bear"],
+                "expected_new_tags": ["+N", "+Der/Dim", "+Px1Sg", "+Pl"],
+                "lemma": "maskwa",
```

Comment:
I'm curious: should the system make the lemma [maskosis](https://itwewina.altlab.app/word/maskosis/?paradigm-size=FULL). Its inflection is different: https://itwewina.altlab.app/search?q=nimaskosisak